src/main/webapp/intrigue-api/schema.graphql

```diff
@@ -277,7 +277,7 @@ type Query {
   user: User
   sources: [Source]
   systemProperties: SystemProperties
-  metacardTypes: [MetacardType]
+  metacardTypes: Json
```

Comment:
:question: Not sure if this is bad practice or not. By treating as a json, we get the ease of accessing attributes through a map, but we lose the ability to only query the attributes we care about.

Reply:
Is there a reason you need it to be a map?

Reply:
It doesn't need to be, but I really hate having to use find every time you want to get information about a particular property

Reply:
I guess my main reason is I just think it's a lot easier to access the information you need by using a map rather than an array

Reply:
I think using a hook around `useQuery` might be a better way to go.

```javascript
const useMetacardTypes = () => {    
  const { metacardTypes } useQuery = (...)    
  return metacardTypes.reduce((types, type) => {    
    types[type.id] = type    
    return types    
  }, {})
}
```

Reply:
I mainly suggest this because the `Json` type should be the last resort to solving a problem.

Reply:
Sweet, I figured there was probably a better alternative. Thanks! :100: